apis/v0/breakglass.go

```diff
@@ -0,0 +1,107 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v0
+
+import (
+	"fmt"
+
+	"github.com/lestrrat-go/jwx/v2/jwa"
+	"github.com/lestrrat-go/jwx/v2/jws"
+	"github.com/lestrrat-go/jwx/v2/jwt"
+)
+
+const (
+	// BreakglassHMACSecret is the HMAC key to use for creating breakglass tokens.
+	// Breakglass tokens are already "unverified", so having this static secret
+	// does not introduce additional risk, and breakglass is disabled by default.
+	BreakglassHMACSecret = "BHzwNUbxcgpNoDfzwzt4Dr2nVXByUCWl1m8Eq2Jh26CGqu8IQ0VdiyjxnCtNahh9" //nolint:gosec
```

Comment:
I've also seen practice to name such variable as "XxxNotSoSecret"

Reply:
But I wonder why use such a complicated string instead of something like "BREAKGLASS"

Reply:
Technically we need at least 64 bytes to avoid leaking. Go doesn't enforce this, but Java does (it throws an exception if you give the HMAC256 less than 64 bytes of key material). We could opt to make it a really long human-readable string, but it would need to be 64 bytes or longer.

Reply:
LOL. Made one but up to you `ABCXYZ-JVS-BREAKGLASS-TOKEN-HMAC-NOT-SO-SECRET-XXXXXXXXXXXXXXXXX`

Reply:
Lol let's stick to the random string.